addon/views/lazy-item.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import StyleBindingsMixin from 'ember-table/mixins/style-bindings';
+
+export default Ember.View.extend(StyleBindingsMixin, {
+  itemIndex: null,
+  prepareContent: Ember.K,
+  teardownContent: Ember.K,
+  rowHeightBinding: 'parentView.rowHeight',
+  styleBindings: ['width', 'top', 'display'],
+
+  top: Ember.computed(function() {
+    return this.get('itemIndex') * this.get('rowHeight');
+  }).property('itemIndex', 'rowHeight'),
+
+  // TODO(azirbel): Add explicit else case
```

Comment:
Based on this comment, did we return something in the `else` case in coffeescript?